src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/prisoner/Events.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner
+
+object Events {
+  const val PRISONER_CREATED = "prisoner-offender-search.prisoner.created"
+  const val PRISONER_UPDATED = "prisoner-offender-search.prisoner.updated"
+}
```

Comment:
These look like test only constants - if so they would be better in the `TEST` tree. I also don't think we need to use an `object` to store strings - they can just be defined in a file